### DISCUSSION TITLE
Submit error

### DISCUSSION THREAD

**Discussion by LuluPrestige** [community/community#54755] [GitHub Education] [answered]
I took my student ID and clicked the "Process my application" button to authenticate my student on GitHub. However, after loading, it became 100%, so the site refreshed and went back to the initial state. It is not even a registration application. Is it server error? please help

**Comment by iamreiyn** [comment, marked as answer]
Hi @PrestigeLulu 

Are you able to try it on a different browser? Firefox, Edge, for example.
Let us know if the problem still persists

**Reply by LuluPrestige** [reply]
I tried it again in edge. And it is worked.
But I was rejected in 1 second

This is reason. But i got them all


Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

**Reply by iamreiyn** [reply]
You will have to translate the documents to English and upload them alongside your main one, and your GitHub profile should match the details of your student ID that you've provided, such as the name.

**Reply by LuluPrestige** [reply]
okay thanks i will translate the documents to Eng
